tests/pyproject.toml

```diff
@@ -8,10 +8,21 @@ build-backend = "scikit_build_core.build"
 [project]
 name = "pybind11_tests"
 version = "0.0.1"
-dependencies = ["pytest", "pytest-timeout", "numpy", "scipy"]
```

Comment:
Was scipy not used anyway?

Reply:
It's optional, and there's not an iOS build of it yet, so figured it was fine to drop it on the Pyodide build for simplicity.

Reply:
Technically, I should make these test groups and load them from cibuildwheel. I could try that.

Reply:
Done